charts/athens-proxy/templates/deployment.yaml

```diff
@@ -23,9 +20,7 @@ spec:
   template:
     metadata:
       labels:
-        app: {{ template "fullname" . }}
-        chart: "{{ .Chart.Name }}-{{ .Chart.Version }}"
-        release: "{{ .Release.Name }}"
+        {{- include "athens.metaLabels" . | nindent 8 }}
```

Comment:
This was the only area without the `heritage` label which is now added. My intention was to keep everything as passive as possible, but i think adding in this extra label here is ok. I can create a separate template helper that doesn't have the heritage label if necessary